infra.bs

```diff
@@ -644,6 +646,35 @@ Standard that should be reported and addressed.
 </div>
 
 
+<h3 id=ecma-262-abstract-operations>ECMA-262 Abstract Operations</h3>
+
```

Comment:
I think you need to ease into the topic...
- ECMA-262 has these things called "abstract operations".
- Algorithms in Infra-based specs can invoke them.
- The style of invoking them is different from how you invoke Infra-based algs.
- Something about the correspondence between Infra values and 262 values.
- Sometimes, the return value is a 'completion record'...